services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no `MovieSession` is found with the given `id`. Consider handling this exception to prevent potential crashes.